model/src/pyrenew/deterministic/nullrv.py

```diff
@@ -46,10 +47,10 @@ def sample(
         Returns
         -------
         tuple
-            Containing None.
+            Containing a SampledValue with None.
         """
 
-        return (None,)
+        return (SampledValue(None),)
 
 
 class NullProcess(NullVariable):
```

Comment:
Is `NullProcess` still used? It seems to be completely identical to `NullVariable`. Artifact of when there were `RandomProcess`es that were separate/distinct from `RandomVariable`s?

Reply:
> Is `NullProcess` still used? It seems to be completely identical to `NullVariable`. Artifact of when there were `RandomProcess`es that were separate/distinct from `RandomVariable`s?

Unsure if this is used right now. The main difference is that `process.*` has the `duration` argument. At some point, the idea of making process, latent, observed, etc. different animals came up. This could add to it.